### PR TITLE
Let's make our code pretty. ;)

### PR DESCRIPTION
**Pull request by surajhell88**
List of changes which this PR represents. 👍 
- Using `prettier` and `.editorconfig` for enforcing certain code style rules while we all contribute to this library.
- Also changed the package name to **@js-geeks/utils**
- Adding the necessary list of filenames to `.gitignore`
- Code style changes according to `prettier` to already existing utils.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org/
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```
Comment (omkarnagvekar) [inline .editorconfig]: As discussed earlier, we can use indent size 4 instead of 2

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# misc
+.DS_Store
+
+npm-debug.log*
```
Comment (omkarnagvekar) [inline .gitignore]: Can we use *.log so that all files are excluded?

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
```
Comment (omkarnagvekar) [inline .prettierrc]: Again here tabWidth can be 4

File: README.md [hunk 4]
```diff
@@ -1,6 +1,58 @@
 # utils
+
 Utility functions that we commonly use from lodash or other utility libraries such as async (https://www.npmjs.com/package/async)
 
-Packages Available : 
+Packages Available :
 
     1. Debounce
+
+## Development
```
Comment (omkarnagvekar) [inline README.md]: Just want to appreciate @surajhell88 for writing the readme in appropriate manner. Keep it up.

File: .editorconfig (hunk 1 above)
Comment (ManishKrShukla) [reply .editorconfig]: indent size 4 is a good idea. Mostly tab based spacing is preferable. Personal opinion though

File: .gitignore (hunk 2 above)
Comment (ManishKrShukla) [reply .gitignore]: Seconded.

File: .prettierrc (hunk 3 above)
Comment (surajhell88) [reply .prettierrc]: # Need to have a vote on it. 😄

File: .gitignore (hunk 2 above)
Comment (surajhell88) [reply .gitignore]: Yes, I guess that can also do.

**Pull request merged**